src/valhalla_run_route.cc

```diff
@@ -206,6 +206,20 @@ const valhalla::TripLeg* PathTest(GraphReader& reader,
     const auto projections = Search(locations, reader, cost);
     std::vector<PathLocation> path_location;
     valhalla::Options options;
+
+    for (const auto& ll : shape) {
```

Comment:
shape is only needed for map matching. regular routing doesnt use it. is this used for map matching?

Reply:
yes.  https://github.com/valhalla/valhalla/blob/master/src/valhalla_run_route.cc#L214